product-service/src/main/java/com/example/productservice/controller/ProductController.java

```diff
@@ -0,0 +1,42 @@
+package com.example.productservice.controller;
+
+import com.example.productservice.dto.ProductRequest;
+import com.example.productservice.dto.ProductResponse;
+import com.example.productservice.service.ProductService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/products")
```

Comment:
https://www.baeldung.com/rest-versioning
Let`s add version of api

Reply:
done